lib/content-services/src/lib/category/categories-management/categories-management.component.scss

```diff
@@ -1,8 +1,14 @@
 .adf-categories-management {
+    padding-top: 12px;
```

Comment:
the component should not enforce the paddings, it's the parent component that defines that if needed. I suggest moving this to `ContentMetadataComponent` that hosts this component